client/galaxy/scripts/apps/panels/admin-panel.js

```diff
@@ -0,0 +1,169 @@
+var _l = require( 'utils/localization' );
```

Comment:
Glad you included this here, but can you also use it to localize the strings below?  We don't need to add translations for them all right now, of course, but the basic infrastructure should be handled as new code/strings are added, IMO.